src/internal/operators/switchMap.ts

```diff
@@ -36,6 +36,16 @@ export function switchMap<T, R, O extends ObservableInput<any>>(project: (value:
  * subsequent inner Observables.
  *
  * ## Example
+ * Generate new Observable according to source Observable values
+ * ```javascript
+ * import { of } from 'rxjs';
+ * import { switchMap } from 'rxjs/operators';
+ *
+ * const switched = of(1,2,3).pipe(switchMap((x: number) => of( x, x ** 2, x ** 3)));
```

Comment:
Can you add space between the numbers `of(1,2,3)`? Not that it's wrong as it is right now but formatting it looks better.

Also there's an extra space here `of( x, ...`. Again, the example is great but this small change will make it look better.

Reply:
add the space in `of(1, 2, 3)`